src/main/cpp/src/api/genomicsdb.cc

```diff
@@ -353,23 +540,39 @@ void GatherVariantCalls::operate_on_columnar_cell(const GenomicsDBColumnarCell&
   contig_position++;
   genomic_interval_t genomic_interval(std::move(contig_name),
                                       std::make_pair(contig_position, contig_position+end_position-coords[1]));
-
   std::vector<genomic_field_t> genomic_fields;
+
+  // jDebug: Need to find out what the right way to get ALT and REF(s) is
+  std::string jDebugRef;
+  std::string jDebugAlt;
+
   // Ignore first field as it is "END"
   for (auto i=1u; i<query_config.get_num_queried_attributes(); i++) {
     if (cell.is_valid(i)) {
       genomic_field_t field_vec(query_config.get_query_attribute_name(i),
                                 cell.get_field_ptr_for_query_idx(i),
                                 cell.get_field_length(i));
       genomic_fields.push_back(field_vec);
+
+      if(query_config.get_query_attribute_name(i) == "REF") {
```

Comment:
I never did figure out how to get the ALT and REF value, so this is extracting them from the genomic fields.

Reply:
@nalinigans - is there a better way for me to get the ALT and REF values here?

Reply:
@nalinigans - since this might be a common operation, it might be good to return a vector<string_view> for ALT (and maybe string_view for REF) [from  cell].